### PR TITLE
create object models and implement them into tests

### PR DESCRIPTION
**Pull request by anezboretskiy**
- [ ] **Breaking change?** (if so, please describe the impact and migration path for existing application instances)

**What changes did you make?** (Give an overview)
create object models and implement them into tests

**Is there anything you'd like reviewers to focus on?**

**How Has This Been Tested?** (put an "x" (case-sensitive!) next to an item)

- [x] No need to
- [ ] Manually (please, describe, if necessary)
- [ ] Unit checks
- [ ] Integration checks
- [ ] Covered by existing automation

**Checklist** (put an "x" (case-sensitive!) next to all the items, otherwise the build will fail)
- [x] I have performed a self-review of my own code
- [x] I have commented my code, particularly in hard-to-understand areas
- [x] I have made corresponding changes to the documentation (e.g. **ENVIRONMENT VARIABLES**)
- [x] My changes generate no new warnings (e.g. Sonar is happy)
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] New and existing unit tests pass locally with my changes
- [x] Any dependent changes have been merged

Check out [Contributing](https://github.com/provectus/kafka-ui/blob/master/CONTRIBUTING.md) and [Code of Conduct](https://github.com/provectus/kafka-ui/blob/master/CODE-OF-CONDUCT.md)

**A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java [hunk 1]
```diff
@@ -11,32 +13,30 @@
...
-        Helpers.INSTANCE.apiHelper.createTopic(CLUSTER_NAME, TOPIC_TO_UPDATE);
-        Helpers.INSTANCE.apiHelper.createTopic(CLUSTER_NAME, TOPIC_TO_DELETE);
-    }
-
-    @AfterAll
-    public static void afterAll() {
-        Helpers.INSTANCE.apiHelper.deleteTopic(CLUSTER_NAME, TOPIC_TO_UPDATE);
-        Helpers.INSTANCE.apiHelper.deleteTopic(CLUSTER_NAME, TOPIC_TO_DELETE);
-        Helpers.INSTANCE.apiHelper.deleteTopic(CLUSTER_NAME, NEW_TOPIC);
+        ApiHelper apiHelper = Helpers.INSTANCE.apiHelper;
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:37]: is this additional line needed? why we can't call this instance inside the lambda?

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java [hunk 2]
```diff
@@ -1,179 +1,178 @@
...
                 .goToSideMenu(CLUSTER_NAME, MainPage.SideMenuOptions.SCHEMA_REGISTRY);
-        pages.schemaRegistry.openSchema(SCHEMA_AVRO_API_UPDATE)
+        pages.schemaRegistry.openSchema(AVRO_API.getName())
                 .waitUntilScreenReady()
                 .openEditSchema();
         Assertions.assertTrue(new SchemaEditView().isSchemaDropDownDisabled(),"isSchemaDropDownDisabled()");
         new SchemaEditView().selectCompatibilityLevelFromDropdown(CompatibilityLevel.CompatibilityEnum.NONE)
-                .setNewSchemaValue(fileToString(PATH_AVRO_FOR_UPDATE))
+                .setNewSchemaValue(fileToString(AVRO_API.getValuePath()))
+                .selectCompatibilityLevelFromDropdown(CompatibilityLevel.CompatibilityEnum.NONE)
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java:76]: why did we add this step?

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java [hunk 3]
```diff
@@ -11,32 +13,30 @@
...
+    private static final String SUITE_TITLE = "Topics";
+    private static final Topic TOPIC_FOR_UPDATE = new Topic()
+            .setName("topic-to-update")
+            .setCompactPolicyValue("Compact")
+            .setTimeToRetainData("604800001")
+            .setMaxSizeOnDisk("20 GB")
+            .setMaxMessageBytes("1000020")
+            .setMessageKey(fileToString(System.getProperty("user.dir") + "/src/test/resources/producedkey.txt"))
+            .setMessageContent(fileToString(System.getProperty("user.dir") + "/src/test/resources/testData.txt"));
+    private static final String TOPIC_NAME_FOR_DELETE = "topic-to-delete";
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java:32]: the goal of this PR is to make test interact with objects. why did you leave constants like this?

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java [hunk 4]
```diff
@@ -12,98 +13,93 @@
...
-    public static final String TOPIC_FOR_UPDATE_CONNECTOR = "topic_for_update_connector";
-    public static final String FIRST_CONNECTOR = "first";
-    public static final String CONNECTOR_FOR_DELETE = "sink_postgres_activities_e2e_checks_for_delete";
-    public static final String CONNECTOR_FOR_UPDATE = "sink_postgres_activities_e2e_checks_for_update";
+    private static final long SUITE_ID = 10;
+    private static final String SUITE_TITLE = "Kafka Connect";
+    private static final String CONNECT_NAME = "first";
+    private static final List <String> TOPIC_NAME_LIST = new ArrayList<>();
+    private static final List <Connector> CONNECTOR_LIST = new ArrayList<>();
+    private static final String TOPIC_FOR_CREATE_CONNECTOR = "topic_for_connector";
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java:27]: is that topic or topic name? i've asked you to refactor naming on create objects here

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java [hunk 5]
```diff
@@ -12,98 +13,93 @@
...
-
-    @AfterAll
-    public static void afterAll() {
-        ApiHelper apiHelper = Helpers.INSTANCE.apiHelper;
-        apiHelper.deleteConnector(CLUSTER_NAME, FIRST_CONNECTOR, SINK_CONNECTOR);
-        apiHelper.deleteConnector(CLUSTER_NAME, FIRST_CONNECTOR, CONNECTOR_FOR_UPDATE);
-        apiHelper.deleteTopic(CLUSTER_NAME, TOPIC_FOR_CONNECTOR);
-        apiHelper.deleteTopic(CLUSTER_NAME, TOPIC_FOR_DELETE_CONNECTOR);
-        apiHelper.deleteTopic(CLUSTER_NAME, TOPIC_FOR_UPDATE_CONNECTOR);
+        String produceMessage = getResourceAsString("message_content_create_topic.json");
```
Comment (VladSenyuta) [inline kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java:40]: is this instance really message?? here we also have an discussion

**Review by VladSenyuta — CHANGES_REQUESTED** [review]

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java (hunk 1 above)
Comment (anezboretskiy) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java]: Refactored

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java (hunk 2 above)
Comment (anezboretskiy) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/SchemasTests.java]: Deleted this step

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java (hunk 3 above)
Comment (anezboretskiy) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/TopicTests.java]: Refactored

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java (hunk 4 above)
Comment (anezboretskiy) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java]: Done

File: kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java (hunk 5 above)
Comment (anezboretskiy) [reply kafka-ui-e2e-checks/src/test/java/com/provectus/kafka/ui/tests/ConnectorsTests.java]: discuss and refactored

**Review by VladSenyuta — APPROVED** [review]

**Pull request merged**